docs/logs/source-generation/Program.cs

```diff
@@ -0,0 +1,42 @@
+// <copyright file="Program.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using Microsoft.Extensions.Logging;
+using OpenTelemetry.Logs;
+
+public class Program
+{
+    public static void Main()
+    {
+        using var loggerFactory = LoggerFactory.Create(builder =>
+        {
+            builder.AddOpenTelemetry(options => options
+                .AddConsoleExporter());
+        });
+
+        var logger = loggerFactory.CreateLogger<Program>();
+
+        logger.FoodPriceChanged("artichoke", 9.99);
+
+        logger.FoodRecallNotice(
+            logLevel: LogLevel.Critical,
+            brandName: "Dole",
```

Comment:
Do we want to use a dummy company? This is not a public repo so it should be fine anyway.

Reply:
I don't know. This information is coming from today's FDA official web site https://www.fda.gov/safety/recalls-market-withdrawals-safety-alerts so I think it's already public?

Reply:
Maybe we can use Contoso or Northwind? (which seems old school) 😃

Reply:
I'm okay with having this company's name here in the example. I was just wondering if there's any best practice or guideline   for calling out actual companies in our example to avoid any legal ramifications. I'm not sure if there is any.

Reply:
> Maybe we can use Contoso or Northwind? (which seems old school) 😃

Yeah we could go with any of the standard fictional companies that are used in Microsoft docs: https://social.technet.microsoft.com/wiki/contents/articles/1117.list-of-fictional-companies-used-in-microsoft-material.aspx

Reply:
Good point, I've changed the name to Contoso.